### DISCUSSION TITLE
til-site/posts/lessons-learned-from-6-months-of-operating-a-teensy-tiny-news-archive/

### DISCUSSION THREAD

**Discussion by giscus** [hiAndrewQuinn/til#16] [Announcements]
# til-site/posts/lessons-learned-from-6-months-of-operating-a-teensy-tiny-news-archive/

The best websites are home-cooked meals. Andrew’s Selkouutiset Archive was birthed after I realized there was no obvious way to fetch the previous articles of the “Easy Finnish” daily news broadcast. This annoyed me as a student of the language. “Here we have a stream”, I thought, “of high-quality, human-written, interesting practice material, and no easy way to access it!” So I went out of my way to create such a way, and me and my language skills have been profiting off of it ever since.

https://hiandrewquinn.github.io/til-site/posts/lessons-learned-from-6-months-of-operating-a-teensy-tiny-news-archive/

**Comment by log101** [comment]
What does the cron jobs look like? I’ve been trying different ways of deploying static sites lately too .)
